docs/mirror.md

```diff
@@ -0,0 +1,175 @@
+
+    ```{.bash data-prompt="$"}
+    $ rsync -avrt rsync://rsync.percona.com/rsync/<repo-name>/apt/pool/main/p/<product_name>/*.<os-version>*.deb /opt/debs/
+    ```
+
+    For example, to download Percona Backup for MongoDB packages on Ubuntu 22.04, the command is the following:
+
+    ```{.bash .no-copy}
+    $ rsync -avrt rsync://rsync.percona.com/rsync/pbm/apt/pool/main/p/percona-backup-mongodb/*.jammy*.deb /opt/debs/
+    ```
+
+    Check the list of available repositories for [MySQL](mysql.md), [MongoDB](mongodb.md), [PostgreSQL](postgresql.md) and [Percona Tools](tools.md).
+
+4. Create the Packages file. This is the file that the package manager uses to retrieve the information about new and updated packages when you run `apt-get update`. Use the `dpkg-scanpackages` command:
+
+    ```{.bash data-prompt="$"}
+    $ cd /opt/debs
+    $ dpkg-scanpackages . /dev/null > Packages
+    ```
+
+5. Add your repository to the `etc/apt/sources.list` file for the package manager to find new or updated packages from your repository:
+   
+    ```{.bash data-prompt="$"}
+    $ echo "deb [trusted=yes] file:///opt/debs ./" | sudo tee -a /etc/apt/sources.list
+    ```
+
+6. Run `apt-get update` to check that the package manager can read from your repository.
+
+
+## On RHEL and derivatives
+
+
+1. Install the `createrepo` utility. It creates yum repositories and their metadata caches
+
+    ```{.bash data-prompt="$"}
+    $ sudo yum install createrepo
+    ```
+
+2. Create the directory where your mirror repository resides. For example, `/opt/rpms`
+
+    ```{.bash data-prompt="$"}
+    $ sudo mkdir /opt/rpms
+    ```
+
+3. Create the repository metadata using `createrepo`
+
+    ```{.bash data-prompt="$"}
+    $ sudo createrepo /opt/rpms
+    ```
+
+4. Download RPM packages from Percona repository to your local mirror using `rsync`:
+
+    ```{.bash data-prompt="$"}
+    $ rsync -avrt rsync://rsync.percona.com/rsync/<repo_name>/yum/release/<os_version>/RPMS/x86_64/ /opt/rpms/
+    ```
+
+    For example, to download packages for Percona Distribution for PostgreSQL 15 on Oracle Linux 9, the command is the following:
+
+    ```{.bash .no-copy}
+    rsync -avrt rsync://rsync.percona.com/rsync/ppg-15/yum/release/9/RPMS/x86_64/ /opt/rpms/
+    ```
+    
+     Check the list of available repositories for [MySQL](mysql.md), [MongoDB](mongodb.md), [PostgreSQL](postgresql.md) and [Percona Tools](tools.md).
+
+5. Update the repository metadata:
+
+    ```{.bash data-prompt="$"}
+    $ sudo createrepo /opt/rpms
+    ```
+
+6. Create the repository configuration file. It must meet the following criteria:
+
+    * It must be located in `/etc/yum.repos.d/` 
+    * It must have the extension `.repo` to be recognized by `yum`.
+
+    The following is the example of the configuration file:
+
+    ```init title="/etc/yum.repos.d/percona-local.repo"
+    [percona-local]
+    name=Local Percona repository
+    baseurl=file:///opt/rpms/
+    gpgcheck=1
```

Comment:
with gpgcheck option it is needed to import gpg key

Reply:
How to do it? Please advise the command

Reply:
curl -s https://raw.githubusercontent.com/percona/percona-repositories/release-1.0-27/rpm/RPM-GPG-KEY-Percona | sudo rpm --import -


or 
wget -qO- https://raw.githubusercontent.com/percona/percona-repositories/release-1.0-27/rpm/RPM-GPG-KEY-Percona | sudo rpm --import -